Makefile

```diff
@@ -102,7 +102,13 @@ reset-installer:
 
 .PHONY: docker-test
 docker-test:
-	@docker compose -f docker-compose.dev.yml up
+	@docker run --rm \
```

Comment:
Weird, I didn't have to do any of this, I just did the docker compose up

Reply:
interesting, it must work if you have composer installed outside of docker, that's what the laravel doc says to do if you don't have composer installed